backend/src/domain/contracts/repos/user.ts

```diff
@@ -1,3 +1,5 @@
+import { Role } from '@/infra/repositories/mysql/entities'
```

Comment:
Tira esse import de Role e as menções dele no output, isso é da `feat/roles-and-permissions`